tests/test_emr/test_emr_boto3.py

```diff
@@ -1045,3 +1045,78 @@ def test_security_configurations():
     ex.value.response["Error"]["Message"].should.match(
         r"Security configuration with name .* does not exist."
     )
+
+
+@mock_emr
+@mock_ec2
```

Comment:
There is a script that tests whether all the required dependencies are included in setup.py, `scripts/int_test.sh`
The short version is:
For each service: create a venv, install dependencies for that service only, and run all the tests

This test will cause the script to fail, as calling `mock_ec2` will require some imports that haven't been installed.

TLDR: Would you mind moving this test to a new file, `test_emr_integration.py`?
In general, all tests that cover multiple services should be declared in a file with that specific name, so the script knows to exclude those when testing the dependencies